homeassistant/components/yale_smart_alarm/config_flow.py

```diff
@@ -140,35 +139,27 @@ async def async_step_user(
         )
 
 
-class YaleOptionsFlowHandler(OptionsFlow):
+class YaleOptionsFlowHandler(OptionsFlowWithConfigEntry):
     """Handle Yale options."""
 
-    def __init__(self, entry: ConfigEntry) -> None:
-        """Initialize Yale options flow."""
-        self.entry = entry
-
     async def async_step_init(
         self, user_input: dict[str, Any] | None = None
     ) -> ConfigFlowResult:
         """Manage Yale options."""
-        errors: dict[str, Any] = {}
 
-        if user_input:
+        if user_input is not None:
```

Comment:
We might need to check this - as in my testing it seems to be initialised with and empty dict - not `None`